mathics/builtin/functional/apply_fns_to_lists.py

```diff
@@ -75,15 +76,14 @@ class Apply(InfixOperator):
         "Heads": "False",
     }
 
-    def eval_invalidlevel(self, f, expr, ls, evaluation, options={}):
-        "Apply[f_, expr_, ls_, OptionsPattern[Apply]]"
-
-        evaluation.message("Apply", "level", ls)
-
     def eval(self, f, expr, ls, evaluation, options={}):
-        """Apply[f_, expr_, Optional[Pattern[ls, _?LevelQ], {0}],
+        """Apply[f_, expr_, Optional[ls_, {0}],
```

Comment:
@rocky this is the place where this `ls` parameter happens. I propose this change because with a small change in the sequence of evaluations, these rules seem to fail, confusing the first option with the `Optional` parameter. This issue often happens when these two patterns are used together. However, with the changes in the last version of the other PRs, it seems I do not hit the issue.

Reply:
Ok. Thanks for the explanation.